client/keys/export.go

```diff
@@ -25,13 +29,31 @@ func ExportKeyCommand() *cobra.Command {
 				return err
 			}
 
-			armored, err := clientCtx.Keyring.ExportPrivKeyArmor(args[0], encryptPassword)
-			if err != nil {
-				return err
+			hex, _ := cmd.Flags().GetBool(flagHex)
```

Comment:
Please rename the variable. It sound that we are doing a Hex serialization, but it is used to decided if we use armored export.

Reply:
renamed to `unarmored`